pom.xml

```diff
@@ -1102,6 +1102,58 @@
             </properties>
         </profile>
 
+        <!-- Profile for testing with HerdDB -->
+        <profile>
+            <id>test-herddb-docker</id>
+            <activation>
+                <property>
+                    <name>test-herddb-docker</name>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.herddb</groupId>
+                    <artifactId>herddb-jdbc</artifactId>
+                    <version>0.22.0</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+            <properties>
+                <connection.driver.name>herddb.jdbc.Driver</connection.driver.name>
```

Comment:
what about a plain java test since herddb is java? docker is mainly an additional test to enable to test which looks not needed here, wdyt?

Reply:
For me docker is also fine. The benefit of a docker image is that you drop it and have a very well defined initial status.

Reply:
the main problem is about adding more jars in the classpath.

I am fine with using in-memory db as well, tests will be probably faster? 

we should have both.

I would start with docker in order to not need to spend time in debugging potential problems about re-initialization of the server

Reply:
I would dependency:unpack the zip/stack and just launch it with a dedicated classloader (no conflict wth openjpa)
Docker is fine but don't expect it to be ran often except on the CI from time to time - this is my main point, only Mark runs docker profiles AFAIK.